### ISSUE TITLE
Unable to view the list of users

### ISSUE THREAD

**Issue by nasli** [owncloud/core#22550]
### Steps to reproduce
1.Create server with today master https://github.com/owncloud/core/commit/8a8209796d4577644228121edc2231ae027217c7
2.Log in with admin user
3.Add an user

### Expected behaviour
View it is reloaded and you can see the new user

### Actual behaviour
Nothing happens, try to refresh browser and a loading appears forever, log out an log in again, go to users view and loading appears forever
Note: the user has been created well

### Server configuration
{"installed":true,"maintenance":false,"version":"9.0.0.12","versionstring":"9.0.0 beta 2","edition":""}

### Logs
Error	no app in context	Exception: {"Exception":"DomainException","Message":"Calendar tables are missing. Nothing to do.","Code":0,"Trace":"#0 \/opt\/owncloud\/apps\/dav\/lib\/migration\/migratecalendars.php(57): OCA\\Dav\\Migration\\CalendarAdapter->setup()\n#1 \/opt\/owncloud\/apps\/dav\/appinfo\/application.php(194): OCA\\Dav\\Migration\\MigrateCalendars->setup()\n#2 \/opt\/owncloud\/apps\/dav\/appinfo\/install.php(27): OCA\\Dav\\AppInfo\\Application->migrateCalendars()\n#3 \/opt\/owncloud\/lib\/private\/installer.php(617): include('\/opt\/owncloud\/a...')\n#4 \/opt\/owncloud\/lib\/private\/installer.php(568): OC_Installer::includeAppScript('\/opt\/owncloud\/a...')\n#5 \/opt\/owncloud\/lib\/private\/installer.php(542): OC_Installer::installShippedApp('dav')\n#6 \/opt\/owncloud\/lib\/private\/setup.php(370): OC_Installer::installShippedApps()\n#7 \/opt\/owncloud\/core\/controller\/setupcontroller.php(64): OC\\Setup->install(Array)\n#8 \/opt\/owncloud\/lib\/base.php(822): OC\\Core\\Controller\\SetupController->run(Array)\n#9 \/opt\/owncloud\/index.php(39): OC::handleRequest()\n#10 {main}","File":"\/opt\/owncloud\/apps\/dav\/lib\/migration\/calendaradapter.php","Line":49}	2016-02-22T08:21:40+00:00

**Comment by LukasReschke** [comment]
Can reproduce.

**Comment by LukasReschke** [comment]
Stacktrace after #22551:

```
{"reqId":"7Z0KX4sY+u5jpYIu9gsQ","remoteAddr":"::1","app":"index","message":"Exception: {\"Exception\":\"OCP\\\\Files\\\\NotFoundException\",\"Message\":\"\\\/test\",\"Code\":0,\"Trace\":\"#0 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/lib\\\/private\\\/avatarmanager.php(85): OC\\\\Files\\\\Node\\\\Root->get('\\\/test')\\n#1 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/settings\\\/controller\\\/userscontroller.php(179): OC\\\\AvatarManager->getAvatar('test')\\n#2 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/settings\\\/controller\\\/userscontroller.php(408): OC\\\\Settings\\\\Controller\\\\UsersController->formatUserForIndex(Object(OC\\\\User\\\\User), Array)\\n#3 [internal function]: OC\\\\Settings\\\\Controller\\\\UsersController->create('test', 'test', Array, '')\\n#4 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/lib\\\/private\\\/appframework\\\/http\\\/dispatcher.php(159): call_user_func_array(Array, Array)\\n#5 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/lib\\\/private\\\/appframework\\\/http\\\/dispatcher.php(89): OC\\\\AppFramework\\\\Http\\\\Dispatcher->executeController(Object(OC\\\\Settings\\\\Controller\\\\UsersController), 'create')\\n#6 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/lib\\\/private\\\/appframework\\\/app.php(110): OC\\\\AppFramework\\\\Http\\\\Dispatcher->dispatch(Object(OC\\\\Settings\\\\Controller\\\\UsersController), 'create')\\n#7 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/lib\\\/private\\\/appframework\\\/routing\\\/routeactionhandler.php(45): OC\\\\AppFramework\\\\App::main('UsersController', 'create', Object(OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer), Array)\\n#8 [internal function]: OC\\\\AppFramework\\\\routing\\\\RouteActionHandler->__invoke(Array)\\n#9 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/lib\\\/private\\\/route\\\/router.php(273): call_user_func(Object(OC\\\\AppFramework\\\\routing\\\\RouteActionHandler), Array)\\n#10 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/lib\\\/base.php(856): OC\\\\Route\\\\Router->match('\\\/settings\\\/users...')\\n#11 \\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/index.php(39): OC::handleRequest()\\n#12 {main}\",\"File\":\"\\\/Users\\\/lukasreschke\\\/Documents\\\/Programming\\\/master\\\/lib\\\/private\\\/files\\\/node\\\/root.php\",\"Line\":181}","level":3,"time":"2016-02-22T08:43:52+00:00"}
```

**Comment by LukasReschke** [comment]
Regression caused by https://github.com/owncloud/core/pull/22410

**Comment by rullzer** [comment]
PR in #22554

**Comment by nasli** [comment]
:+1: fixed on #22554
